beacon_node/beacon_chain/tests/payload_invalidation.rs

```diff
@@ -1050,7 +1050,7 @@ async fn invalid_parent() {
 
     // Ensure the block built atop an invalid payload is invalid for gossip.
     assert!(matches!(
-        rig.harness.chain.clone().verify_block_for_gossip(block.clone()).await,
+        rig.harness.chain.clone().verify_block_for_gossip(block.clone(), 0).await,
```

Comment:
Same as earlier comment about CGC in gossip verification tests - be good to make input as close to prod as possible.

Reply:
2ba26543e5f039b1ee9774bb1c4c2f21dfa2a5e1